ingenialink/ethercat/network.py

```diff
@@ -303,7 +397,7 @@ def disconnect_from_slave(self, servo: EthercatServo) -> None:  # type: ignore [
         self.servos.remove(servo)
         if not self.servos:
             self.stop_status_listener()
-            self._ecat_master.close()
+            self.close_ecat_master()
```

Comment:
We would still be missing a no_gil lock that is waited for all gil operations are done